test/test_gnss.py

```diff
@@ -40,11 +38,11 @@ def temp_file():
 
 
 def test_getDateTime():
-    f1 = '20080101T060000'
-    f2 = '20080101T560000'
-    f3 = '20080101T0600000'
-    f4 = '20080101_060000'
-    f5 = '2008-01-01T06:00:00'
+    f1 = Path('20080101T060000')
```

Comment:
@garlic-os I don't think these should be Paths, aren't they string inputs?

Reply:
These are actually file names, and getDateTime does path operations on them. In usual execution, these will be GPS Zenith delay CSV files with a timestamp like this in their full filename.